src/test/a11y/a11y.ts

```diff
@@ -25,19 +27,29 @@ const routesNotTested = [
   '/mock-login',
   '/logout',
   '/robots.txt',
+  '/file-publication',
+  '/list-publication',
 ];
 
 const rawDataCourt = fs.readFileSync(path.resolve(__dirname, '../unit/mocks/courtAndHearings.json'), 'utf-8');
 const rawDataLive = fs.readFileSync(path.resolve(__dirname, '../unit/mocks/liveCaseStatusUpdates.json'), 'utf-8');
 const rawDataCaseEventGlossary = fs.readFileSync(path.resolve(__dirname, '../unit/mocks/CaseEventGlossary.json'), 'utf-8');
 const rawSJPData = fs.readFileSync(path.resolve(__dirname, '../unit/mocks/trimmedSJPCases.json'), 'utf-8');
-
+const onePubJsonData = fs.readFileSync(path.resolve(__dirname, '../unit/mocks/onePublicationJson.json'), 'utf-8');
+const mockPDF = new Blob(['testPDF']);
+const indivPubJsonObject = {'data':mockPDF};
+const onePubJson = JSON.parse(onePubJsonData);
 const allCourtData = JSON.parse(rawDataCourt);
 const courtData = allCourtData[0];
 const liveCaseData = JSON.parse(rawDataLive).results;
 const caseEventGlossaryData = JSON.parse(rawDataCaseEventGlossary);
 const sjpCases = JSON.parse(rawSJPData).results;
 
+sinon.stub(SummaryOfPublicationsRequests.prototype, 'getListOfPubs').returns(onePubJson);
+sinon.stub(SummaryOfPublicationsRequests.prototype, 'getIndividualPubMetadata').returns(onePubJson[0]);
+sinon.stub(SummaryOfPublicationsRequests.prototype, 'getIndividualPubFile').resolves(indivPubJsonObject);
```

Comment:
Don't believe lines 49 -> 51 are needed now that file/list are excluded from these tests

Potentially same with 39 -> 41

Reply:
sorted